tracing-futures/Cargo.toml

```diff
@@ -22,7 +22,7 @@ default = ["std-future"]
 futures-01 = ["futures_01"]
 futures-03 = ["std-future", "futures", "futures-task"]
 std-future = ["pin-project"]
-tokio-alpha = ["std-future", "tokio_02"]
+tokio_02 = ["std-future", "tokio"]
```

Comment:
This (and `tokio_01`) should be name like the `futures` features -- using `-` not `_`.

Reply:
@jonhoo the `tokio_02` naming was already there, although we should probably change it. I think it's fine to break that in `tracing-futures` 0.2